notebooks/403-action-recognition-webcam/403-action-recognition-webcam.ipynb

```diff
@@ -0,0 +1,646 @@
+    "model_weights_path_encoder = f\"model/intel/{model_name}/{model_name}-encoder/{precision}/{model_name}-encoder.bin\"\n",
+    "\n",
+    "# Testing the location of *.bin and *.xml\n",
+    "files = os.listdir(f\"model/intel/{model_name}/{model_name}-encoder/{precision}\")\n",
+    "print(files)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Load your labels\n",
+    "\n",
+    "We are using [Kinetics-400 dataset](https://deepmind.com/research/open-source/kinetics), and also we are providing you with the text file embedded into this notebook."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# If you want to use another model, you should change the labels file\n",
+    "labels = \"data/kinetics.txt\"\n",
+    "\n",
+    "if labels:\n",
+    "    with open(labels) as f:\n",
+    "        labels = [line.strip() for line in f]\n",
+    "else:\n",
+    "    labels = None\n",
+    "    \n",
+    "#print(labels[0], np.shape(labels))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
+   "source": [
+    "### Load the models\n",
+    "\n",
+    "We will load the two models for this particular architecture, Encoder and Decoder. Downloaded models are located in a fixed structure, indicating vendor, model name, and precision.\n",
+    "\n",
+    " 1. Initialize inference engine (IECore)\n",
+    " 2. Read the network from *.bin and *.xml files (weights and architecture)\n",
+    " 3. Load the model on the \"CPU.\"\n",
+    " 4. Get input and output names of nodes.\n",
+    "\n",
+    "Only a few lines of code are required to run the model. Let's see it."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Model Initialization function"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#Model initialization\n",
+    "\n",
+    "# initialize inference engine\n",
+    "ie_core = ie.IECore()\n",
+    "\n",
+    "def model_init(model: str, weights: str) -> dict:\n",
+    "    \"\"\"\n",
+    "    Read the network and weights from file, load the \n",
+    "    model on the CPU and get input and output names of nodes\n",
+    "    \n",
+    "    :param: model: model architecture path *.xml\n",
+    "    :param: model: model weights path *.bin\n",
+    "    :retuns: \n",
+    "            exec_net: Encoder model network\n",
+    "            input_key: Input node network\n",
+    "            output_key: Output node network\n",
+    "    \"\"\"\n",
+    "    \n",
+    "    # read the network and corresponding weights from file\n",
+    "    net = ie_core.read_network(model, weights)\n",
+    "    # load the model on the CPU (you can use GPU or MYRIAD as well)\n",
```

Comment:
Have you tested the model on MYRIAD? Not all models work well there and there can be some issues with loading MYRIAD models.

Reply:
No, I have not tested the model on MYRIAD. I will delete that.